app/editor/src/env.json

```diff
@@ -0,0 +1,8 @@
+{
+  "environments": [
```

Comment:
Now how do you tell if it's dev or test?

Reply:
Updated, and yeah if using array then will use the index which will be based on the value in this case, which is quite clear.